src/transformers/dynamic_module_utils.py

```diff
@@ -116,6 +116,9 @@ def check_imports(filename):
     with open(filename, "r", encoding="utf-8") as f:
         content = f.read()
 
+    # filter out try/except block so in custom code we can have try/except imports
+    content = re.sub(r"\s*try\s*:\s*.+\s*except\s*:", "", content, flags=re.MULTILINE)
```

Comment:
```suggestion
    content = re.sub(r"\s*try\s*:\s*.*?\s*except\s*:", "", content, flags=re.MULTILINE)
```
This is to make sure it matches the shortest span, so we don't suppress a lot of code if there are several try except blocks 😅